### PR TITLE
get more data from twitter if access_token is provided 

### PR DESCRIPTION
**Pull request by keul**
This close #116

### CODE REVIEW

**Comment by mmerickel** [comment]
Before I do my own research, can you explain to me how the access_token and access_token_secret are different from the already-computed `credentials['oauthAccessToken']` and `credentials['oauthAccessTokenSecret']`?

**Comment by keul** [comment]
Mmmhh... I need to check. Maybe I'm wrong.

**Comment by keul** [comment]
@mmerickel  you were totally right: already computed token was working! The code now is a lot simpler!

File: velruse/providers/twitter.py [hunk 1]
```diff
@@ -132,6 +133,26 @@ def callback(self, request):
         }]
         profile['displayName'] = access_token['screen_name']
 
+        oauth = OAuth1(
+            self.consumer_key,
+            client_secret=self.consumer_secret,
+            resource_owner_key=access_token['oauth_token'],
+            resource_owner_secret=access_token['oauth_token_secret'],
+            verifier=verifier)
```
Comment (mmerickel) [inline velruse/providers/twitter.py]: The verifier is only necessary for getting the access token, you can leave it off here.

File: velruse/providers/twitter.py [hunk 2]
```diff
@@ -132,6 +133,26 @@ def callback(self, request):
...
+        resp = requests.get(DATA_URL % access_token['screen_name'], auth=oauth)
+        if resp.status_code == 200:
+            # replace display name with the full name, and take additional data
+            profile_data = json.loads(resp.content)
```
Comment (mmerickel) [inline velruse/providers/twitter.py]: requests has a way to convert the body to json already, so you can drop the import and use `resp.json()`.

**Comment by mmerickel** [comment]
If you can make those couple updates I'll merge your work, thanks so much!

**Comment by keul** [comment]
@mmerickel now should be ok!

**Comment by mmerickel** [comment]
Thanks again for the PR!

**Pull request merged**